message-schemas/package.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
I don't get why this is a nested directory/package - can we just promote this to the top level? IMO the new way of doing things should deprecate everything else. You could still leave the old generated ros messages in here for now if needed, but we can plan to remove them in the future.

Reply:
Yeah I was trying to avoid interfering with existing scripts & generated files. I think that's why I started with it in a nested package.